cdap-docs/_common/common-build.sh

```diff
@@ -60,6 +60,7 @@ EOF`
 
 SCRIPT=`basename ${0}`
 SCRIPT_PATH=`pwd`
```

Comment:
Added a *check_starting_directory* function to **build.sh** to check and see that it is started from the correct directory.
Lower-level build scripts already fail if they are not started from the correct directory.